src/components/CMSImage/sanityImageBuilder.ts

```diff
@@ -10,11 +10,13 @@ import getBrowserConfig from "../../browser-lib/getBrowserConfig";
  * Provide a "client like" object instead of using the
  * actual sanity client to cut down on dependency size
  */
-export const sanityClientLike: SanityClientLike = {
-  clientConfig: {
-    projectId: getBrowserConfig("sanityProjectId"),
-    dataset: getBrowserConfig("sanityDataset"),
-    apiHost: `https://${getBrowserConfig("sanityAssetCDNHost")}`,
+export const sanityClientLike: SanityModernClientLike = {
+  config: () => {
+    return {
+      projectId: getBrowserConfig("sanityProjectId"),
+      dataset: getBrowserConfig("sanityDataset"),
+      apiHost: `https://${getBrowserConfig("sanityAssetCDNHost")}`,
+    };
```

Comment:
[This PR](https://github.com/lorenzodejong/next-sanity-image/pull/62) seems to re-simplify this shape, although it hasn't been responded to yet by the package author